ts/src/test/test.ts

```diff
@@ -400,9 +400,9 @@ export default class testMainClass extends baseMainTestClass {
                 }
             }
             // we don't throw exception for public-tests, see comments under 'testSafe' method
-            const failedMsg = errors ? errors.join (', ') : undefined;
             let errorsInMessage = '';
-            if (errors !== undefined) {
+            if (errors) {
```

Comment:
because `errors` could have been an empty array, causing to bypass `!== undefined` check, leading to errors